.github/actions/slack/action.yml

```diff
@@ -17,4 +17,4 @@ runs:
         SLACK_MESSAGE: ${{ inputs.message }}
         SLACK_ICON: https://asap-misc.s3.eu-west-1.amazonaws.com/robot.png
         SLACK_USERNAME: ASAP robot
-        SLACK_COLOR: ${{ job.status }}
+        SLACK_COLOR: ${{ inputs.color || job.status }}
```

Comment:
wasnt the point of this to resolve the color from the status? like if we just g ive it a color as input surely it wont depend on the outcome of the previous job no?